homework-2/utube/src/App.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import './App.css';
+import {PreviewList} from "./components/list_of_previews";
+import {SearchBar} from "./components/search_bar";
+import Search from 'youtube-search';
+import {ActiveVideo} from "./components/active_video";
+
+class App extends Component {
+  constructor(){
+    super();
+    this.state = {
+      previews: [],
+      searchOpts: {
```

Comment:
searchOpts is not a part of a state, they could be static and refers only to API